tests/functional/adapter/materialization/test_view_security.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from dbt.tests.util import run_dbt
+
+from tests.functional.adapter.materialization.fixtures import (
+    seed_csv,
+    model_sql,
+    profile_yml,
+)
+
+
+class TestViewSecurity:
+    """
```

Comment:
I'm assuming that this is copy-paste from another source.
Assuming that the dbt testing framework is here to stay, we don't need to add this introductory comments.

I'd like however to have a comment about *WHAT* is being actually tested and not *HOW* it is tested (the answer to this question is done by the code itself).

Reply:
Good catch, I will improve comments.